### PR TITLE
[FIX] website: prevent error on edit mode save for restricted editor

### PR DESCRIPTION
**Pull request by sga-odoo**
**SPECIFICATION**
Enter in edit mode as website restrictor user, and click on save button, you will get the traceback
    https://youtu.be/OR3m5brmHnE

**LINKS**
TaskID: 2379101
Closes: https://github.com/odoo/odoo/pull/66357

### CODE REVIEW

File: addons/website/static/src/js/editor/wysiwyg_multizone.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ var WysiwygMultizone = Wysiwyg.extend({
                 .then(() => this.editor.save(false))
                 .then(() => ({isDirty: true}));
         } else {
-            return {isDirty: false};
+            return Promise.resolve({isDirty:false});
```
Comment (rdeodoo) [inline addons/website/static/src/js/editor/wysiwyg_multizone.js:114]:
```suggestion
            return Promise.resolve({isDirty: false});
```

**Review by rdeodoo — APPROVED** [review]
I fixed the typo and rebased, also rewrote the commit message to follow convention (this commit will.. / task-id)
Fix working as expected

**Review by qsm-odoo — COMMENTED** [review]
@robodoo delegate=@rdeodoo

**Comment by JKE-be** [comment]
@robodoo delegate=rdeodoo

**Comment by rdeodoo** [comment]
@robodoo r+

**Pull request closed without merging**